libs/designer-ui/src/lib/panel/recommendationpanel/searchResult.tsx

```diff
@@ -152,7 +152,7 @@ export const OperationActionDataFromOperation = (operation: DiscoveryOperation<D
   connectorName: operation.properties.api.displayName,
   category: getConnectorCategoryString(operation.properties.api),
   isTrigger: !!operation.properties?.trigger,
-  isBuiltIn: isBuiltInConnector(operation.properties.api),
+  isBuiltIn: isBuiltInConnector(operation.properties.api) || isPremiumConnector(operation.properties.api),
```

Comment:
keep these seperate, this could cause break/confusion when premium isn't a thing. Do not overload built in.